.github/workflows/canister-tests.yml

```diff
@@ -770,6 +770,13 @@ jobs:
       - name: Release notes
         run: cat ${{ steps.prepare-release.outputs.notes-file }}
 
+      # Create a sha file for dfx pull, referenced by the metadata set in the build
+      - name: Create sha for dfx pull
+        run: |
+          shasum -a 256 \
+            ./internet_identity_dev.wasm.gz \
```

Comment:
```suggestion
            ./internet_identity_dev.wasm.gz > \
```

Reply:
The comment above this code block still says that:
```
          # We do not include a checksum file URL because the II canister is very small and would not
          # benefit much from skipping a canister download in case of existing canister with same
          # checksum.
```
Since you want to remove the `wasm_hash_url` later, it sounds okay to keep the comment. But it is still an inconsistent description.